source/opt/eliminate_dead_input_components_pass.cpp

```diff
@@ -25,20 +25,32 @@
 
 namespace spvtools {
 namespace opt {
+
 namespace {
+
```

Comment:
consistency with other files, no spacing around those namespaces and before

Reply:
Done.